tests/slow/src/test/scala/tests/feature/ImportMissingSymbolCrossLspSuite.scala

```diff
@@ -41,4 +42,85 @@ class ImportMissingSymbolCrossLspSuite
       )
     } yield ()
   }
+
+  test("scala3-extension-import-from-deps") {
+    val path = "b/src/main/scala/x/B.scala"
+    for {
+      _ <- initialize(
+        s"""|/metals.json
+            |{
+            |  "a":{"scalaVersion" : "${V.scala3}"},
+            |  "b":{
+            |    "scalaVersion" : "${V.scala3}",
+            |    "dependsOn": ["a"]
+            |  }
+            |}
+            |/a/src/main/scala/example/A.scala
+            |package example
+            |object IntEnrichment:
+            |  extension (num: Int)
+            |    def incr = num + 1
+            |/$path
+            |package x
+            |def main =
+            |  println(1.incr)
+            |""".stripMargin
+      )
+      _ <- server.didOpen(path)
+      _ <- server.assertCodeAction(
+        path,
+        s"""|package x
+            |def main =
+            |  println(1.<<incr>>)
```

Comment:
Could we maybe add it to the code actions Scala 3 suite? Or at least it would be nice to see the code action applied to test if it works correctly